samples/subsys/usb/hid/src/main.c

```diff
@@ -99,10 +99,17 @@ static void idle_cb(u16_t report_id)
 	LOG_DBG("Idle callback: wrote %d bytes with ret %d", wrote, ret);
 }
 
+static void protocol_cb(u8_t protocol)
+{
+	LOG_DBG("New protocol: %s", (protocol == HID_PROTOCOL_BOOT ?
+				     "boot" : "report"));
```

Comment:
I do not think this is right indentation.

Reply:
Do you need here braces?

Reply:
Removed braces, `"boot"` is aligned to `protocol`.

Reply:
Should it be aligned like:
```
-       LOG_DBG("New protocol: %s", (protocol == HID_PROTOCOL_BOOT ?
-                                    "boot" : "report"));
+       LOG_DBG("New protocol: %s", protocol == HID_PROTOCOL_BOOT ?
+               "boot" : "report");
```

Reply:
I believe your editor should align it right.

Reply:
Fixed.